MapboxNavigation/RouteVoiceController.swift

```diff
@@ -369,7 +369,7 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         }
         
         // Only localized languages will have a proper fallback voice
-        utterance.voice = AVSpeechSynthesisVoice(language: Bundle.main.preferredLocalizations.first)
+        utterance.voice = AVSpeechSynthesisVoice(language: Locale.preferredLanguages.first!)
```

Comment:
`en` should be good enough for AVSpeechSynthesizer. If it isn’t specific enough for Polly, use `Locale.current.regionCode`. (`regionCode` is only available on iOS 10+, but you can use `Locale.object(forKey:)` with `NSLocale.Key.countryCode` to get the same information.